### PR TITLE
Use documentation.js 3.0.0

### PR DESCRIPTION
**Pull request by jfirebaugh**
This is pending a fix for event documentation. Existing documentation for events looks like:

image

This was possible because documentation.js had invented a separate "event" scope for events. This was contrary to JSDoc, which treats events as standard static or (usually) instance scope, and documentation 3.0 dutifully follows this convention, placing events back into the `instance` and `static` arrays. But I'd still like to use the ⓔ sigil for events -- I think `ⓔ click` is a lot clearer than `#click`, which looks like an instance method that simulates a click.

Options:

* Add a preinstalled `sigil` handlebars helper to documentation.js HTML output code that does what I want. Doesn't feel like a good solution because it's a core documentation.js change for one specific use case.
* Switch from handlebars to a non-logic-less template engine (or a pluggable meta-engine like [consolidate](https://www.npmjs.com/package/consolidate)).
* ?

cc @tmcw @lucaswoj

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,7 +15,7 @@
     "feature-filter": "^1.0.2",
     "geojson-vt": "^2.1.0",
     "gl-matrix": "^2.3.1",
-    "glify": "^0.5.0",
+    "glify": "^0.5.1",
```
Comment (lucaswoj) [inline package.json]: Was this intended?

File: docs/_generate/generate.js [hunk 2]
```diff
@@ -1,24 +0,0 @@
...
-    .pipe(html({
-        name: pkg.name,
-        version: pkg.version,
-        path: __dirname
-    }))
-    .pipe(through(function (file) {
-        if (file.path === 'index.html') {
-            fs.writeFileSync(path.join(__dirname, '../_posts/3400-01-01-api.html'), file.contents);
-        }
-    }));
```
Comment (lucaswoj) [inline docs/_generate/generate.js]: Love the red! :+1:

**Comment by lucaswoj** [comment]
Other than the seemingly-spurious `glify` dependency change, this looks good to :shipit:.

File: package.json (hunk 1 above)
Comment (jfirebaugh) [inline package.json]: Yeah, this pulls in https://github.com/mapbox/glify/commit/d5a972aed8d26984c60cab73e31b4c589591b7c9 which was necessary to get glify to play nice with documentation.js's CLI.

**Comment by lucaswoj** [comment]
Handlebars isn't strictly logic-less -- it does include `if` blocks. Is that not enough to render events differently?

**Comment by lucaswoj** [comment]
We might be able to include a more semantic `isEvent` helper instead, like

```
{{#isEvent kind}}
```

**Comment by jfirebaugh** [comment]
Good point, it so happens that only events have an `event` property, so `{{#if event}}...{{else}}...{{/if}}` will work.

**Pull request closed without merging**